hw/bsp/nucleo-f072rb/bsp.yml

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+bsp.arch: cortex_m0
+bsp.compiler: compiler/arm-none-eabi-m0
+bsp.linkerscript:
+    - "hw/bsp/nucleo-f072rb/nucleo-f072rb.ld"
+    - "@apache-mynewt-core/hw/mcu/stm/stm32f0xx/stm32f0xx.ld"
+bsp.downloadscript: "hw/bsp/nucleo-f072rb/nucleo-f072rb_download.sh"
+bsp.debugscript: "hw/bsp/nucleo-f072rb/nucleo-f072rb_debug.sh"
+bsp.downloadscript.WINDOWS.OVERWRITE: "hw/bsp/nucleo-f072rb/nucleo-f072rb_download.cmd"
+bsp.debugscript.WINDOWS.OVERWRITE: "hw/bsp/nucleo-f072rb/nucleo-f072rb_debug.cmd"
+
+bsp.flash_map:
+    areas:
+        # System areas.
+        FLASH_AREA_BOOTLOADER:
```

Comment:
Is it possible to just leave out BOOTLOADER/IMAGE_1 areas from here?

Reply:
Some dependencies require the definitions that are generated by newt, eg, flash_map requires `FLASH_AREA_IMAGE_1` to exist, also some to bootloader; removing it would require me to change my sample programs' deps...